distributed/tests/test_spill.py

```diff
@@ -1,11 +1,11 @@
 import pytest
 
-pytest.importorskip("zict")
-
 from dask.sizeof import sizeof
 
 from distributed.spill import SpillBuffer
 
+pytest.importorskip("zict")
+
```

Comment:
We can remove this altogether since `zict` is a requirement for `distributed` :tada:

```suggestion
```